ignite/pkg/nodetime/programs/swagger-combine/swagger-combine.go

```diff
@@ -75,6 +75,11 @@ func (c *Config) AddSpec(id, path string) error {
 	return nil
 }
 
+//// CombineDir combines openapi files from a folder and save into the out path.
+//func CombineDir(ctx context.Context, dir, out string) error {
+//
+//}
+
```

Comment:
Should this commented code be removed? Or is it pending to be implemented?

```suggestion
```

Reply:
Ohhh, sorry, It was here by mistake